independent-publisher-connectors/Dexcom/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[YOUR_VALUE]",
+          "scopes": [
+            "offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://api.dexcom.com/v2/oauth2/login"
+            },
+            "tokenUrl": {
+              "value": "https://api.dexcom.com/v2/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://api.dexcom.com/v2/oauth2/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#59A618",
```

Comment:
@Flow-Joe Please update it to `"#da3b01"` which is the default iconBrandColor for all IP connectors.

Reply:
Updated